atlasdb-client/src/test/java/com/palantir/atlasdb/util/TimedOnlyInstrumentationFilterTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.assertj.core.api.AbstractBooleanAssert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.palantir.atlasdb.metrics.Timed;
+
+@RunWith(MockitoJUnitRunner.class)
+public final class TimedOnlyInstrumentationFilterTest {
+
+    @Mock
+    private TestService testService;
+    private final TimedOnlyInstrumentationFilter filter = new TimedOnlyInstrumentationFilter();
```

Comment:
nit: move the final field above the non-final

Reply:
Done